net.fasterland.converseen.json

```diff
@@ -0,0 +1,62 @@
+{
+    "id": "net.fasterland.converseen",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "converseen",
+    "rename-desktop-file": "converseen.desktop",
+    "rename-appdata-file": "converseen.appdata.xml",
+    "finish-args": [
+        "--device=dri",
+        "--filesystem=home",
```

Comment:
What in particular does this need in the filesystem that it doesn't get from the portal?

Reply:
This is indeed not required.